cove/locale/es/LC_MESSAGES/django.po

```diff
@@ -1032,3 +1037,139 @@ msgstr ""
 "Los datos que usted quería explorar ya no existen.\n"
 "\n"
 "Esto se debe a que todos los datos suministrados a este sitio web se borran automáticamente después de 7 días, y por lo tanto el análisis de esos datos ya no está disponible."
+
+#: cove_iati/lib/iati.py:27
+msgid ""
+"We think you tried to upload a XML file, but it is not well formed XML.\n"
+"\n"
+"<span class=\"glyphicon glyphicon-exclamation-sign\" aria-hidden=\"true\"></span> <strong>Error message:</strong> {}"
+msgstr ""
+"Creemos que ha tratado de subir un archivo XML, pero el archivo no es XML "
+"bien formado. <span class=\"glyphicon glyphicon-exclamation-sign\" aria-"
+"hidden=\"true\"></span><strong>Mensaje del Error:</strong>{}"
+
+#: cove_iati/templates/cove_iati/base.html:28
+msgid "Bootstrap theme "
+msgstr "Tema Bootstrap"
+
+#: cove_iati/templates/cove_iati/base.html:28
+msgid ""
+"\n"
+"by Bootswatch"
+msgstr ""
+"\n"
+"creado por Bootswatch"
+
+#: cove_iati/templates/cove_iati/base.html:49
+msgid "How to use "
+msgstr "Uso"
+
+#: cove_iati/templates/cove_iati/explore.html:29
+msgid "A file was "
+msgstr "Un archivo ha sido"
+
+#: cove_iati/templates/cove_iati/explore.html:31
+msgid "downloaded from "
+msgstr "Descargado desde"
+
+#: cove_iati/templates/cove_iati/explore.html:33
+msgid "uploaded "
+msgstr "Subido "
+
+#: cove_iati/templates/cove_iati/explore.html:35
+msgid "on "
+msgstr "en"
```

Comment:
@Bjwebb I realise that the right translation for this _on_  in this context is probably _el_. Can we change it just here or do we need to go to transifex, change and pull it again? I guess I need to do it in transifex otherwise _en_ will be pulled again next time around? Or we can just leave like this too, no big deal :)